generics/abstract.js

```diff
@@ -14,6 +14,10 @@ let Abstract = class Abstract {
   
   constructor(schema) {
     database.createModel(schemas[schema]);
+    if ( schemas[schema].runIndex && schemas[schema].runIndex.length > 0 ) {
+      database.runCompoundIndex(schemas[schema].name,schemas[schema].runIndex);
```

Comment:
@priyanka-TL does this mean we will try to create index every time the model is loaded? say on restarting the node service?

Reply:
yes